dart/lib/src/noop_hub.dart

```diff
@@ -58,5 +58,5 @@ class NoOpHub implements Hub {
   SentryId get lastEventId => SentryId.empty();
 
   @override
-  void addBreadcrumb(Breadcrumb crumb, {dynamic hint}) {}
+  void addBreadcrumb(Breadcrumb? crumb, {dynamic hint}) {}
```

Comment:
```suggestion
  void addBreadcrumb(Breadcrumb crumb, {dynamic hint}) {}
```

it does not make sense to be nullable, people would need to create a `Breadcrumb` at runtime or using the addBreadcrumb(String)